src/ProjectTemplates/Microsoft.Extensions.AI.Templates/Microsoft.Extensions.AI.Templates.csproj

```diff
@@ -21,17 +21,27 @@
     <EnableDefaultItems>false</EnableDefaultItems>
     <NoDefaultExcludes>true</NoDefaultExcludes>
     <NoPackageAnalysis>true</NoPackageAnalysis>
+    <NoWarn>$(NoWarn);NU1702</NoWarn>
```

Comment:
Nit: Add a comment for why NU1702 needs to be ignored

Reply:
For my understanding, why does this project need to target `netstandard2.0`?

Reply:
I'm not totally sure. @SteveSandersonMS, do you know why we target `netstandard2.0`?

Reply:
The starting point was copied from another template package though I don't recall which one. But seeing that others like Aspire's template and [ASP.NET Core ones](https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/Microsoft.DotNet.Web.ProjectTemplates.csproj#L4) just use `$(DefaultNetCoreTargetFramework)`, it looks as if we could and should do the same here.

Reply:
Resolving as I've now removed the `netstandard2.0` target framework.